ansible/group_vars/all.example

```diff
@@ -12,8 +12,8 @@ mysql_root_db_user: root
 mysql_root_db_pass: root
 
 # NginX Server Root
-nginx_server_root: /vagrant
-nginx_try_files: /index.php$is_args$args
+nginx_server_root: /vagrant/iaptus/html
```

Comment:
This is intended, this is a public repo there should be no reference to `iaptus`.